nucypher/cli/worklock.py

```diff
@@ -0,0 +1,204 @@
+
+
+def _setup_emitter(general_config):
+    emitter = general_config.emitter
+    emitter.clear()
+    emitter.banner(WORKLOCK_BANNER)
+    return emitter
+
+
+class WorkLockOptions:
+
+    __option_name__ = 'worklock_options'
+
+    def __init__(self, bidder_address: str):
+        self.bidder_address = bidder_address
+
+    def create_agent(self, registry):
+        agent = ContractAgency.get_agent(WorkLockAgent, registry=registry)
+        return agent
+
+
+group_worklock_options = group_options(
+    WorkLockOptions,
+    bidder_address=option_bidder_address)
+
+
+@click.group()
+def worklock():
+    """
+    Manage stakes and other staker-related operations.
+    """
+    pass
+
+
+@worklock.command(name='status')
+@group_registry_options
+@group_worklock_options
+@group_general_config
+def status(general_config, registry_options, worklock_options):
+    emitter = _setup_emitter(general_config)
+    registry = registry_options.get_registry(emitter, general_config.debug)
+    paint_worklock_status(emitter=emitter, registry=registry)
+    if worklock_options.bidder_address:
+        paint_worklock_participant_status(emitter=emitter,
+                                          registry=registry,
+                                          bidder_address=worklock_options.bidder_address)
+    return  # Exit
+
+
+@worklock.command(name='bid')
+@option_force
+@group_registry_options
+@group_worklock_options
+@group_general_config
+@click.option('--value', help="Eth value of bid", type=click.INT)
+def bid(general_config, worklock_options, registry_options, force, value):
+    emitter = _setup_emitter(general_config)
+    if not value:
+        value = int(Web3.fromWei(click.prompt("Enter bid amount in ETH", type=click.FloatRange(min=0)), 'wei'))
+        if force:
+            raise click.MissingParameter("Missing --value.")
+    if not worklock_options.bidder_address:
+        worklock_options.bidder_address = select_client_account(emitter=emitter,
+                                                                provider_uri=general_config.provider_uri)
+    registry = registry_options.get_registry(emitter, general_config.debug)
+    worklock_agent = worklock_options.create_agent(registry=registry)
+
+    if not force:
+        paint_worklock_participant_notice(emitter=emitter, bidder_address=worklock_options.bidder_address, registry=registry)
+        click.confirm(f"Place WorkLock bid of {Web3.fromWei(value, 'ether')} ETH?", abort=True)
+
+    receipt = worklock_agent.bid(bidder_address=worklock_options.bidder_address, value=value)
+    emitter.message("Publishing WorkLock Bid...")
+    paint_receipt_summary(receipt=receipt, emitter=emitter, chain_name=worklock_agent.blockchain.client.chain_name)
+    return  # Exit
+
+
+@worklock.command(name='cancel-bid')
+@group_registry_options
+@group_worklock_options
+@group_general_config
+def burn_unclaimed_tokens(general_config, registry_options, worklock_options):
```

Comment:
Did you intend to have two functions with names `burn_unclaimed_tokens` but which are matched with two different commands `cancel-bid` and `burn-unclaimed-tokens`?

I assume not, and this function's name was intended to be `cancel_bid`?

```suggestion
def cancel_bid(general_config, registry_options, worklock_options):
```

Reply:
:heavy_check_mark: